demos/crossroad_camera_demo/main.cpp

```diff
@@ -578,22 +571,19 @@ int main(int argc, char *argv[]) {
         auto total_t0 = std::chrono::high_resolution_clock::now();
         slog::info << "Start inference " << slog::endl;
 
+        cv::Mat frame = cap->read();
+        if (!frame.data) throw std::logic_error("Can't read an image from the input");
+
+        cv::Size graphSize{frame.cols / 4, 60};
+        Presenter presenter(FLAGS_u, frame.rows - graphSize.height - 10, graphSize);
```

Comment:
Won't this create a problem if the frame size starts changing?

Reply:
It will but we already have that problem in tests when demos are executed on imageNet.